src/java/org/apache/cassandra/metrics/AccordMetrics.java

```diff
@@ -0,0 +1,279 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.metrics;
+
+import java.lang.reflect.Field;
+import java.util.concurrent.TimeUnit;
+
+import accord.api.EventsListener;
+import accord.local.Command;
+import accord.primitives.Deps;
+import accord.primitives.PartialDeps;
+import accord.primitives.Timestamp;
+import accord.primitives.TxnId;
+import com.codahale.metrics.Counting;
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.Timer;
+import org.apache.cassandra.service.accord.AccordService;
+
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
+
+public class AccordMetrics
+{
+    public final static AccordMetrics readMetrics = new AccordMetrics("ro");
+    public final static AccordMetrics writeMetrics = new AccordMetrics("rw");
+
+    /**
+     * The time between start on the coordinator and commit on this replica.
+     */
+    public final Timer commitLatency;
+
+    /**
+     * The time between start on the coordinator and execution on this replica.
+     */
+    public final Timer executeLatency;
+
+    /**
+     * The time between start on the coordinator and application on this replica.
+     */
+    public final Timer applyLatency;
+
+    /**
+     * Duration of applying changes.
+     */
+    public final Timer applyDuration;
+
+    /**
+     * The number of dependencies of the transaction on this replica.
+     */
+    public final Histogram deps;
+
+    public final Meter progressLogSize;
+
+    /**
+     * The number of dependencies of the transaction.
+     */
+    public final Histogram coordinatorDeps;
+
```

Comment:
We have a handful of metrics in the following section, and there are a couple general things:

1.) I'm not sure we need the `Trx` suffixes. They don't seem to add much information?

2.) The concepts are all coordinator-level concepts, AFAICT, so the `coordinator` prefix doesn't seem necessary? (i.e. There's no concept of a "replica slow path"...)

Reply:
1) Ok
2) It is just for the ease of use - I believe that having them grouped under distinct categories makes it easier to navigate in simple JMX clients. However, I'm not adamantly opposed to changing that if you wish

Reply:
I like the naming scheme overall after the last couple commits, FWIW